python/ray/experimental/collective/tensor_transport_manager.py

```diff
@@ -31,6 +31,15 @@ def is_one_sided() -> bool:
             bool: True if the backend is one-sided, False otherwise.
         """
 
+    @abstractmethod
+    @abstractmethod
+    def can_abort_transport(self) -> bool:
+        """Whether the backend can abort the transport.
```

Comment:
```suggestion
        """Whether the backend can abort the transport.
        
        If this returns False, then Ray will kill involved actors upon system errors to avoid hanging.
```